src/Standards/Generic/Sniffs/Operators/StrictComparisonSniff.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Ensures that constant names are all uppercase.
+ *
+ * @author    Vincent Langlet <vincentlanglet@exemple.com>
+ * @copyright 2006-2015 Squiz Pty Ltd (ABN 77 084 670 600)
+ * @license   https://github.com/squizlabs/PHP_CodeSniffer/blob/master/licence.txt BSD Licence
+ */
+
+namespace PHP_CodeSniffer\Standards\Generic\Sniffs\Operators;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+class StrictComparisonSniff implements Sniff
+{
+
+    /**
+     * Types to replace: key is operator to replace, value is operator to replace with.
+     *
+     * @var array
+     */
+    public $operators = [
+        T_IS_EQUAL     => '===',
```

Comment:
Using the constant as the key will make this property nearly impossible to use as the value of the token constants may change between PHP versions.

Suggestion: use the type instead, i.e `'T_IS_EQUAL'` (as a string) to allow for setting this from a ruleset in a reliable manner.

Reply:
How can I get the code in the register method from the type then ?

Reply:
You mean this ?

```php
$targets = [];
foreach ($this->operators as $type => $replacement) {
    $targets[] = constant($type);
}

return $targets;
```

Reply:
Yes, thanks, I didn't know the constant function.